### PR TITLE
Changelog for v48.23.0

### PR DESCRIPTION
**Pull request by brpowell**
### What does this PR do?

Adds the changelog for v48.23.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
+# 48.23.0 - July 2, 2020
```
Comment (brpowell) [inline packages/salesforcedx-vscode/CHANGELOG.md:1]: @ekapner Changelog for Thursday's release

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Added 8 new apex code snippets ([PR #2262](https://github.com/forcedotcom/salesforcedx-vscode/pull/2262), [Issue #2261](https://github.com/forcedotcom/salesforcedx-vscode/issues/2261)) - Contribution by [lukecotter](https://github.com/lukecotter)
```
Comment (ekapner) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: Technically, Apex should be capitalized.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Fixed an issue where cancelling a command threw an error ([PR #2280](https://github.com/forcedotcom/salesforcedx-vscode/pull/2280))
+
+#### salesforcedx-vscode-lwc
+
+- Fixed an issue with the local dev server start and component preview commands failing to run due to encoding issues ([PR #2287](https://github.com/forcedotcom/salesforcedx-vscode/pull/2287), [Issue #2284](https://github.com/forcedotcom/salesforcedx-vscode/issues/2284))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:25]:
```suggestion
- Fix `SFDX: Start Local Development Server` and `SFDX: Preview Component Locally` failing to open a browser window due to URL encoding issues ([PR #2287](https://github.com/forcedotcom/salesforcedx-vscode/pull/2287), [Issue #2284](https://github.com/forcedotcom/salesforcedx-vscode/issues/2284))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,29 @@
...
+- Added 8 new Apex code snippets ([PR #2262](https://github.com/forcedotcom/salesforcedx-vscode/pull/2262), [Issue #2261](https://github.com/forcedotcom/salesforcedx-vscode/issues/2261)) - Contribution by [lukecotter](https://github.com/lukecotter)
+
+#### salesforcedx-vscode-core
+
+- Added tooltips to the org picker and org status bar items ([PR #2272](https://github.com/forcedotcom/salesforcedx-vscode/pull/2272))
+
+#### salesforcedx-vscode-lwc
+
+- Added `SFDX: Stop Watching All Lightning Web Component Tests` command ([PR #2298](https://github.com/forcedotcom/salesforcedx-vscode/pull/2298))
+
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md:16]: @brpowell , remove Added before each line (it's in the Added section) and rephrase as:
Eight new Apex code snippets
Tooltips for Org Picker and Status Bar items
New `SFDX: Stop Watching All Lightning Web Component Tests` command

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,29 @@
...
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Fixed an issue where cancelling a command threw an error ([PR #2280](https://github.com/forcedotcom/salesforcedx-vscode/pull/2280))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md:21]: @brpowell, rephrase as:
Fix SFDX command throwing an error when canceled.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,29 @@
+# 48.23.0 - July 2, 2020
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Added 8 new Apex code snippets ([PR #2262](https://github.com/forcedotcom/salesforcedx-vscode/pull/2262), [Issue #2261](https://github.com/forcedotcom/salesforcedx-vscode/issues/2261)) - Contribution by [lukecotter](https://github.com/lukecotter)
```
Comment (brpowell) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]:
```suggestion
- Eight new Apex code snippets ([PR #2262](https://github.com/forcedotcom/salesforcedx-vscode/pull/2262), [Issue #2261](https://github.com/forcedotcom/salesforcedx-vscode/issues/2261)) - Contribution by [lukecotter](https://github.com/lukecotter)
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,29 @@
...
+
+#### salesforcedx-vscode-core
+
+- Added tooltips to the org picker and org status bar items ([PR #2272](https://github.com/forcedotcom/salesforcedx-vscode/pull/2272))
```
Comment (brpowell) [inline packages/salesforcedx-vscode/CHANGELOG.md:11]:
```suggestion
- Tooltips for Org Picker and Open Org status bar items ([PR #2272](https://github.com/forcedotcom/salesforcedx-vscode/pull/2272))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,29 @@
...
+
+#### salesforcedx-vscode-lwc
+
+- Added `SFDX: Stop Watching All Lightning Web Component Tests` command ([PR #2298](https://github.com/forcedotcom/salesforcedx-vscode/pull/2298))
```
Comment (brpowell) [inline packages/salesforcedx-vscode/CHANGELOG.md:15]:
```suggestion
- New `SFDX: Stop Watching All Lightning Web Component Tests` command ([PR #2298](https://github.com/forcedotcom/salesforcedx-vscode/pull/2298))
```

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 5 above)
Comment (brpowell) [inline packages/salesforcedx-vscode/CHANGELOG.md:21]:
```suggestion
- Fix SFDX command throwing an error when cancelled ([PR #2280](https://github.com/forcedotcom/salesforcedx-vscode/pull/2280))
```

**Pull request merged**